projects/1.18/assets/industrial-foregoing/industrialforegoing/lang/zh_cn.json

```
@@ -497,22 +519,22 @@
   "tooltip.industrialforegoing.mob_crusher.produce": "生产精华液",
   "tooltip.industrialforegoing.mob_crusher.produce_extra": "生物会产生精华液",
   "tooltip.industrialforegoing.mob_crusher.consume": "转换为抢夺模式",
-  "tooltip.industrialforegoing.mob_crusher.consume_extra_1": "生物不会产生精华，但每次击杀",
-  "tooltip.industrialforegoing.mob_crusher.consume_extra_2": "都有一定几率应用随机等级的抢夺效果",
+  "tooltip.industrialforegoing.mob_crusher.consume_extra_1": "生物不会产生精华，但每次击杀都会",
+  "tooltip.industrialforegoing.mob_crusher.consume_extra_2": "在任何级别都有概率抢夺生效",
   "tooltip.industrialforegoing.bl.compact_unit": "显示具体数量",
-  "tooltip.industrialforegoing.bl.stack_unit": "显示组数",
+  "tooltip.industrialforegoing.bl.stack_unit": "显示堆叠的组数",
   "tooltip.industrialforegoing.bl.void_unit": "清除多出来的物品",
   "tooltip.industrialforegoing.bl.no_void_unit": "不清除多出来的物品",
   "tooltip.industrialforegoing.bl.show_unit_display": "显示储存的东西",
   "tooltip.industrialforegoing.bl.hide_unit_display": "隐藏储存的东西",
   "tooltip.industrialforegoing.backpack.void": "清除：",
   "tooltip.industrialforegoing.backpack.refill": "补充：",
-  "tooltip.industrialforegoing.backpack.magnet_enabled": "开启磁引",
-  "tooltip.industrialforegoing.backpack.magnet_disabled": "关闭磁引",
-  "tooltip.industrialforegoing.backpack.pickup_all": "开启物品&经验自动拾取",
-  "tooltip.industrialforegoing.backpack.item_pickup_enabled": "开启物品自动拾取",
-  "tooltip.industrialforegoing.backpack.xp_pickup_enabled": "开启经验自动拾取",
-  "tooltip.industrialforegoing.backpack.pickup_disabled": "关闭自动拾取",
+  "tooltip.industrialforegoing.backpack.magnet_enabled": "启用磁铁",
+  "tooltip.industrialforegoing.backpack.magnet_disabled": "禁用磁铁",
+  "tooltip.industrialforegoing.backpack.pickup_all": "物品 & XP 自动提取启用",
```

```suggestion
  "tooltip.industrialforegoing.backpack.pickup_all": "物品&XP自动提取启用",
```
这地方不用加空格了力（但如果是特殊情况那这个suggestion可以无视）

---

这个空格改了估计整个文档都要改一次......如果不嫌麻烦的话也可以改